Sources/Purchasing/Purchases/PurchasesType.swift

```diff
@@ -679,6 +679,28 @@ public protocol PurchasesType: AnyObject {
 
     #if os(iOS) || VISION_OS
```

Comment:
Was the `presentCodeRedemptionSheet` function moved inside this conditional intentionally? I think this might be a breaking change since it changes where the function is compiled (even if unavailable) so you cannot use runtime availability checks with it.